lib/BannerPlugin.js

```diff
@@ -13,30 +13,43 @@ const validateOptions = require("schema-utils");
 const schema = require("../schemas/plugins/BannerPlugin.json");
 
 const wrapComment = str => {
-	if (!str.includes("\n")) return Template.toComment(str);
-	return `/*!\n * ${str
+	let trimmedStr = str.trim();
```

Comment:
Why did you add trimming? This makes the PR a breaking change.

Reply:
Fixed.